BuildResidentialScheduleFile/resources/schedules.csv

```diff
@@ -0,0 +1,10 @@
+Schedule Name,Element,Values,Data Source
+lighting_interior,LightingInteriorMonthlyMultipliers,"1.075, 1.064951905, 1.0375, 1.0, 0.9625, 0.935048095, 0.925, 0.935048095, 0.9625, 1.0, 1.0375, 1.064951905",
+dishwasher,HotWaterDishwasherMonthlyMultiplier,"1.083, 1.056, 1.023, 0.999, 0.975, 0.944, 0.918, 0.928, 0.938, 0.984, 1.059, 1.094",
+clothes_washer,HotWaterClothesWasherMonthlyMultiplier,"0.968, 1.013, 0.99, 1.034, 1.019, 1.015, 1.048, 1, 1.021, 0.949, 0.945, 0.999",
+clothes_dryer,ClothesDryerMonthlyMultiplier,"1.09, 1.054, 1.044, 0.996, 0.992, 0.967, 0.931, 0.906, 0.923, 0.955, 1.035, 1.108",
+cooking_range,CookingMonthlyMultiplier,"1.038, 1.026, 0.976, 0.945, 0.965, 0.947, 0.939, 0.965, 0.967, 1.006, 1.098, 1.129",
```

Comment:
Can we get these data sources filled in?

Reply:
Perhaps some clues in this file? https://github.com/NREL/OpenStudio-HPXML/blob/master/BuildResidentialScheduleFile/resources/schedules_config.md